tests/unit/cluster/atomic-slot-migration.tcl

```diff
@@ -597,12 +600,16 @@ start_cluster 3 3 {tags {external:skip cluster} overrides {cluster-node-timeout
         wait_for_condition 1000 10 {
             [S 0 mem_slot_migration_output_buffer] > 1000000
         } else {
-            fail "ait for buffer to accumulate on source side (more than 1m)"
+            fail "Failed to wait for buffer to accumulate on source side (more than 1m)"
         }
 
         # After some time, the client output buffer limit should be reached
         wait_for_log_messages 0 {"*Client * closed * for overcoming of output buffer limits.*"} $loglines 1000 10
-        assert_match {*send*stream*} [migration_status 0 $task_id last_error]
+        wait_for_condition 1000 10 {
+            [string match {*send*stream*} [migration_status 0 $task_id last_error]]
+        } else {
+            fail "ASM task did not fail as expected"
+        }
```

Comment:
lol I have the exact change for this test. Though, it still failed: https://github.com/tezc/redis/actions/runs/18634196346/job/53123086548

It became a lot more unlikely but still failed (once in 6 dailys)